src/cryptoadvance/specter/templates/includes/helpers.jinja

```diff
@@ -1,26 +1,189 @@
 <!-- This file contains javascript helper functions that also can use jinja -->
 <script type="text/javascript">
 
-    // Formats an unitLabel (e.g. from a tx output) to a useful unit label
-    function formatUnitLabel(unitLabel, convertToSat){
-        var newLabel = unitLabel;
+
+
+    /*
+        Formats the btc amount as sats. 
+        Args:
+            value (Union[float, str]): (in BTC) Will convert string to float. 
+            enableDigitFormatting (bool, optional): Will group the Satoshis into blocks of 3,
+                e.g. 3 123 456, and color the blocks. Defaults to false.
+        Returns:
+            str: The formatted btc amount as html code.
+        */
+    function internalFormatBtcAmountAsSats(
+        value,
+        enableDigitFormatting=false,
+    ) {
+        const locale = 'en-US';
+        if (isNaN(parseFloat(value))){
+            return parseFloat(value)        
+        }
+        var s = (parseFloat(value) * 1e8).toLocaleString(locale, { 
+            minimumFractionDigits: 0, 
```

Comment:
What's the purpose of the options? `minimumFractionDigits` and `maximumFractionDigits`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString   How many digits are there  after the decimal separator.

Reply:
Got it. Probably the defaults are 0 anyways? But, isn't bad to have it explicit here.